.travis.yml

```diff
@@ -4,5 +4,6 @@ go:
   - 1.6
 
 script:
-  - go build
-  - go test ./...
+  - make init
```

Comment:
To specify what I meant: of course you can keep `make test` in the travis script, but I would remove the govendor dependency from the test target as it works without it, too